woocommerce/woocommerce-gutenberg-products-block
# Remove ProductPreview component

It looks like we have a `ProductPreview` component that has been dead code for some months now. It was initially used by the product grid blocks, but when those were migrated to Server Side Rendering they stopped relying on it (see #565 for more details).

### How to test the changes in this Pull Request:

1. Make sure `npm run start` and `npm run build` still work and do some smoke testing.
